cleancat/base.py

```diff
@@ -491,12 +492,19 @@ class Enum(Choices):
     """
     Like Choices, but expects a Python 3 Enum.
     """
+    def __init__(self, enum_cls, choices=None, **kwargs):
```

Comment:
Should add a docstring since it's a bit confusing if you don't follow the code.

Reply:
Another way would be to allow you to do `Enum([MyChoices.A, MyChoices.B])`, and derive `enum_cls` via `choices[0].__class__` if `inspect.isclass(choices)` returned false. That way you wouldn't need two arguments and you wouldn't have to specify the class type explicitly.